include/crocoddyl/multibody/costs/contact-cop-position.hxx

```diff
@@ -0,0 +1,92 @@
+///////////////////////////////////////////////////////////////////////////////
+// BSD 3-Clause License
+//
+// Copyright (C) 2020, University of Duisburg-Essen, University of Edinburgh
+// Copyright note valid unless otherwise stated in individual files.
+// All rights reserved.
+///////////////////////////////////////////////////////////////////////////////
+
+#include "crocoddyl/core/utils/exception.hpp"
+#include "crocoddyl/multibody/costs/contact-cop-position.hpp"
+
+namespace crocoddyl {
+
+template<typename _Scalar>
+CostModelContactCoPPositionTpl<_Scalar>::CostModelContactCoPPositionTpl(boost::shared_ptr<StateMultibody> state,
```

Comment:
Can you please make sure here that the default activation is quadraticbarrier?

Reply:
I've added `typedef ActivationModelQuadraticBarrierTpl<Scalar> ActivationModelQuadraticBarrier;` and replaced the activation everywhere. Are there additionally requirements for the `cost-base.hpp` (related compilation error occurs)?

Reply:
I've changed it to `support_region` as suggested by Carlos.

Reply:
@julesser you need to create a constructor that doesn't include `activation` argument, and then to pass to `Base` your default one.

If you don't know how to do it, I could deal myself with it.

You will also need to declare both constructors for the Python bindings.